ComponentKit/Utilities/CKEqualityHashHelpers.h

```diff
@@ -102,7 +101,7 @@ namespace CKTupleOperations
       size_t prev = _hash_helper<Tuple, Index-1>::hash(tuple);
       using TypeForIndex = typename std::tuple_element<Index,Tuple>::type;
       size_t thisHash = CK::hash<TypeForIndex>()(std::get<Index>(tuple));
-      return CKHashCombine(prev, thisHash);
+      return CKHash64ToNative(CKHashCombine(prev, thisHash));
```

Comment:
`CKHashCombine` returns a `uint64_t` hash yet we're expected to return a `size_t` hash. This isn't particular safe for 32-bit devices so I try and downscale here if necessary.